rocAL/rocAL/include/rali_api_types.h

```diff
@@ -42,13 +42,34 @@ typedef void * RaliContext;
 typedef void * RaliImage;
 typedef void * RaliMetaData;
 
+typedef std::vector<int> ImageIDBatch,AnnotationIDBatch;
+typedef std::vector<std::string> ImagePathBatch;
+typedef std::vector<float> ScoreBatch,RotationBatch;
+typedef std::vector<std::vector<float>> CenterBatch, ScaleBatch;
+typedef std::vector<std::vector<std::vector<float>>> JointsBatch, JointsVisibilityBatch;
+
 struct TimingInfo
 {
     long long unsigned load_time;
     long long unsigned decode_time;
     long long unsigned process_time;
     long long unsigned transfer_time;
 };
+
+//HRNet training expects meta data (joints_data) in below format, so added here as a type for exposing to user
+struct RaliJointsData
```

Comment:
please modify the structure to represent one batch and pass a list of JointsData structure to represent batch

Reply:
HRNet training expects each member of JointsDataBatch as a vector instead of a vector of JointsData structure due to which we have arrived at the above format